Zend/tests/final_properties/unset/declared_unset_error.phpt

```diff
@@ -0,0 +1,39 @@
+--TEST--
+Test that declared final properties can't be unset
+--FILE--
+<?php
+
+class Foo
+{
+    final public string $property1 = "foo";
+    final public string $property2;
+    final protected string $property3 = "foo";
+
+    public function unsetProperty3(): void
+    {
+        unset($this->property3);
+    }
+}
+
+$foo = new Foo();
+
+try {
+    unset($foo->property1);
+} catch (Error $exception) {
+    echo $exception->getMessage() . "\n";
+}
+
+// Uninitialized typed properties can't be unset
```

Comment:
can't -> can?

Reply:
It is just an outdated text before I discovered how unsetting actually works. ^^ I'm deleting it, no need for it now.